tests/Operation/RoundingTest.php

```diff
@@ -8,21 +8,22 @@
 
 namespace PrestaShop\Decimal\Test\Operation;
 
-use PrestaShop\Decimal\Operation\Rounding;
+use InvalidArgumentException;
+use PHPUnit\Framework\TestCase;
 use PrestaShop\Decimal\DecimalNumber;
+use PrestaShop\Decimal\Operation\Rounding;
 
-class RoundingTest extends \PHPUnit_Framework_TestCase
+class RoundingTest extends TestCase
 {
-
     /**
      * Given decimal number
      * When rounding it using an undefined rounding mode
      * An InvalidArgumentException should be thrown
-     *
-     * @expectedException \InvalidArgumentException
      */
     public function testItThrowsExceptionIfRoundingModeIsInvalid()
     {
+        $this->expectException(InvalidArgumentException::class);
```

Comment:
Agreed. I prefer assertions rather than phpDoc magic annotation

Reply:
The magic annotation has been deprecated and no longer work in the next release, that's the reason ^^

Reply:
I prefer the latest trends always